FlowCrypt/Controllers/Settings/Backup/Backups Option Scene/BackupOptionsViewController.swift

```diff
@@ -103,19 +106,26 @@ extension BackupOptionsViewController {
 
     private func backupToInbox() {
         showSpinner()
-        backupService.backupToInbox(keys: backups, for: userId)
-            .then(on: .main) { [weak self] in
+        cancellable = Just(backups)
```

Comment:
would it be better to use 
`backupService.backupToInbox(keys: input, for: userId)` and operate with this publisher directly instead of `Just(backups)`?

Reply:
No, because you need to tell Combine that all chain should run on global queue. If you run `backupService.backupToInbox()` first, it will run on current thread.

Reply:
please check comments bellow